test_automation/conftest.py

```diff
@@ -0,0 +1,137 @@
+    sys.stderr = plogger.stream_to_logger(logging.getLogger("STDERR"), logging.WARN)
+
+    sig_obj, worker_obj, private_key, server_uri = initialize_objects(config, remainder)
+    worker_obj, uri_client, err_cd = worker_lookup_retrieve(config, worker_obj, server_uri)
+
+    return worker_obj, sig_obj, uri_client, private_key, err_cd
+
+def initialize_objects(config, args):
+    """ Function to initialize common objects for tests. """
+
+    logger.info('***************** INTEL TRUSTED COMPUTE FRAMEWORK (TCF)*****************')
+
+    # static server_uri and private_key of client
+    server_uri = r'http://localhost:1947'
+    private_key = enclave_helper.generate_signing_keys()
+
+    # Initializing Signature object, Worker Object
+    sig_obj = signature.ClientSignature()
+    worker_obj = worker.WorkerDetails()
+
+    # Log computed objects
+    logger.info("server_uri: %s", server_uri)
+    logger.info("config: %s", config)
+    logger.info("sig_obj: %s", sig_obj)
+    logger.info("worker_obj: %s", worker_obj)
+
+    return sig_obj, worker_obj, private_key, server_uri
+
+def worker_lookup_retrieve(config, worker_obj, server_uri):
+    """ Function for computing worker lookup and retrieve once per session. """
+
+    if not server_uri:
+        logger.error("Server URI is not provided")
+        exit(1)
+
+    # logger.info("Execute work order")
+    uri_client = GenericServiceClient(server_uri)
+    response = None
+    wo_id = None
+    request = 1
+
+    err_cd = 0
+    #----------------------------------------------------------------------------------
+    # create worker lookup request
+    output_json_file_name = 'worker_lookup'
+    input_worker_look_up = '{"jsonrpc": "2.0", "method": "WorkerLookUp", "id": 1, "params": {"workerType": 1}}'
+    input_json_str1 = json.dumps(json.loads(input_worker_look_up))
+
+    logger.info("------------------Testing WorkerLookUp------------------")
+
+    # submit worker lookup request and retrieve response
+    logger.info("**********Received Request*********\n%s\n", input_json_str1)
+    response = process_request(uri_client, input_json_str1, output_json_file_name)
+    logger.info("**********Received Response*********\n%s\n", response)
+
+    # check worker lookup response
+    if "result" in response and "totalCount" in response["result"].keys():
+        if response["result"]["totalCount"] == 0:
+            err_cd = 1
+            logger.info("ERROR: Failed at WorkerLookUp - No Workers exist to process workorder.")
+
+    if err_cd == 0:
+        # create worker retrieve request
+        input_worker_retrieve = '{"jsonrpc": "2.0", "method": "WorkerRetrieve", "id": 2, "params": {"workerId": ""}}'
+        input_json_str1 = json.dumps(json.loads(input_worker_retrieve))
+        logger.info("------------------Testing WorkerRetrieve------------------")
+        # Retrieving the worker id from the "WorkerLookUp" response and update the worker id information for the further json requests
```

Comment:
This line is too long and is hard to read. Please limit lines to 80 characters.

Reply:
Fixed line length to below 80